pkgs/development/compilers/rust/rustc.nix

```diff
@@ -299,8 +299,9 @@ stdenv.mkDerivation (finalAttrs: {
         runHook preInstall
 
         python ./x.py --keep-stage=0 --stage=1 install library/std
-        mkdir -v $out/bin $doc $man
+        mkdir -v $out/{bin,libexec} $doc $man
         ln -s ${rustc.unwrapped}/bin/{rustc,rustdoc} $out/bin
+        ln -s ${rustc.unwrapped}/libexec/rust-analyzer-proc-macro-srv $out/libexec
```

Comment:
Should we just link the whole libexec directory?

Reply:
sure, why not